static/app/views/issueList/searchBar.spec.tsx

```diff
@@ -165,4 +166,68 @@ describe('IssueListSearchBar', function () {
       );
     });
   });
+
+  describe('Tags and Fields', function () {
+    it('displays the correct options for the is tag', async function () {
+      MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/tags/',
+        body: [],
+      });
+
+      defaultProps.organization.features = ['issue-stream-search-query-builder'];
+
+      render(<IssueListSearchBar {...defaultProps} />, {
+        router,
+      });
+
+      await userEvent.click(screen.getByRole('combobox', {name: 'Add a search term'}));
+      await userEvent.paste('is:', {delay: null});
+      await userEvent.click(
+        screen.getByRole('button', {name: 'Edit value for filter: is'})
```

Comment:
You might want to change this to a `await screen.findByRole(...)`. This might seem unnecessary since tests are passing, but for a complex component like this, it can lead to flaky tests in CI if we look for something without waiting.